gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/RubricPreviewPanel.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2003-2018 The Apereo Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://opensource.org/licenses/ecl2
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sakaiproject.gradebookng.tool.panels;
+
+import java.util.Map;
+import java.util.List;
+
+import org.apache.wicket.AttributeModifier;
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.extensions.ajax.markup.html.modal.ModalWindow;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.apache.wicket.markup.head.IHeaderResponse;
+import org.apache.wicket.markup.head.StringHeaderItem;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.WebMarkupContainer;
+import org.apache.wicket.markup.html.form.Form;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.ResourceModel;
+import org.apache.wicket.model.StringResourceModel;
+
+import org.sakaiproject.gradebookng.business.GradebookNgBusinessService;
+import org.sakaiproject.gradebookng.business.model.GbGradeInfo;
+import org.sakaiproject.gradebookng.business.model.GbUser;
+import org.sakaiproject.gradebookng.tool.component.GbAjaxButton;
+import org.sakaiproject.portal.util.PortalUtils;
+import org.sakaiproject.rubrics.logic.RubricsConstants;
+import org.sakaiproject.service.gradebook.shared.Assignment;
+
+public class RubricPreviewPanel extends BasePanel {
+
+    @SpringBean(name = "org.sakaiproject.gradebookng.business.GradebookNgBusinessService")
+    private GradebookNgBusinessService businessService;
+
+    private final ModalWindow window;
+
+    public RubricPreviewPanel(final String id, final IModel<Map<String, Object>> model, final ModalWindow window) {
```

Comment:
There is no reason to use a Map here when there is only one value. Just make it IModel<Long> instead.

Reply:
Changed this and it works fine.